buildshiprun/handler.go

```diff
@@ -38,7 +38,7 @@ func Handle(req []byte) string {
 		Source: "buildshiprun",
 	}
 
-	serviceValue := fmt.Sprintf("%s-%s", event.Owner, event.Service)
+	serviceValue := strings.ToLower(fmt.Sprintf("%s-%s", event.Owner, event.Service))
```

Comment:
We don't need to change the service name, this is unrelated to GitHub usernames.